apps/desktop/README.md

```
@@ -0,0 +1,71 @@
+This package is the **OS-level security boundary** for the rewrite. Everything app-related (receipts, projections, broker state, OAuth tokens) lives behind a separate process the renderer reaches over loopback HTTP. The shell only exposes OS verbs (open external URL, show file in folder, app version, broker liveness).
+
+## Run it
+
+```bash
+bun install                    # at repo root, once
+bun run desktop:dev            # boots Electron window
+```
+
+The window shows a single status pane:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)